pyserini/demo/miracl.py

```diff
@@ -70,20 +79,45 @@ def search():
             "miracl.html", search_results=search_results, query=query, lang=lang, retriever=retriever
         )
 
+    @app.route("/lang", methods=["GET"])
```

Comment:
hi @ehsk - I just noticed - we use single quotes throughout the repo. Would you mind changing throughout?

Reply:
Fixed!